src/org.ts

```diff
@@ -9,3 +12,11 @@ export type OrgMemberInfo = {
 export type OrgIdToOrgMemberInfo = {
     [orgId: string]: OrgMemberInfo
 }
+
+export const setActiveOrgId = (orgId: string) => {
+    Cookies.set(ACTIVE_ORG_ID_COOKIE_NAME, orgId)
```

Comment:
Could you also add secure and same-site: lax?

Reply:
Added